client/modules/ftml-wasm/pkg/ftml.d.ts

```diff
@@ -0,0 +1,263 @@
+/* tslint:disable */
```

Comment:
Maybe rename `pkg` -> `vendor` for clarity?

Reply:
That does make more sense. Let me do a rebase and I'll make that change with it.